### PR TITLE
feat: readme batcher cli simplify sending proofs process by using cargo install

### LINKED ISSUE #278: ( README - Batcher CLI) Simplify sending proofs process by using cargo install
To avoid moving so much around folders, we could ask the user to install the CLI and run the commands with it

### PR DESCRIPTION
**Pull request by NicolasRampoldi**
# Description

- Changed `cargo run` for `cargo install` in the batcher client.
To test, run `make install_batcher_client` and then run: 

```bash
batcher-client --proving_system Groth16Bn254 \
--proof <path_to_repo>/batcher/client/test_files/groth16/ineq_1_groth16.proof \
--public_input <path_to_repo>/batcher/client/test_files/groth16/ineq_1_groth16.pub \
--vk <path_to_repo>/batcher/client/test_files/groth16/ineq_1_groth16.vk \
--proof_generator_addr 0x66f9664f97F2b50F62D13eA064982f936dE76657
```

Modifying the `<path_to_repo>` to the corresponding one.

### CODE REVIEW

**Review by jotabulacios — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -28,94 +28,86 @@ Aligned works with EigenLayer to leverage ethereum consensus mechanism for ZK pr
...
 ```bash
-pushd batcher/client/ ; \
-cargo run -- \
+batcher-client \
 --proving_system SP1 \
 --proof test_files/sp1/sp1_fibonacci.proof \
 --vm_program test_files/sp1/sp1_fibonacci-elf \
---conn wss://batcher.alignedlayer.com ;
-popd
+--conn wss://batcher.alignedlayer.com
```
Comment (entropidelic) [inline README.md:59]: should the `proof_generator_addr` be added here too?

File: README.md [hunk 2]
```diff
@@ -28,94 +28,86 @@ Aligned works with EigenLayer to leverage ethereum consensus mechanism for ZK pr
...
```
Comment (entropidelic) [inline README.md:67]: same here

File: README.md (hunk 1 above)
Comment (NicolasRampoldi) [reply README.md:59]: It was removed in this [commit](https://github.com/yetanotherco/aligned_layer/pull/283)

File: README.md (hunk 2 above)
Comment (NicolasRampoldi) [reply README.md:67]: It was removed in this [commit](https://github.com/yetanotherco/aligned_layer/pull/283)

File: README.md (hunk 1 above)
Comment (entropidelic) [reply README.md:59]: oh I see. Now it is an optional parameter. Not relevant to this PR but that should probably be mentioned somewhere in the README

**Review by entropidelic — APPROVED** [review]

**Pull request merged**